### ISSUE TITLE
HTML Special Chars

### ISSUE THREAD

**Issue by martoio** [gloc-mike/prj-rev-bwfs-dasmoto#1]
Taking this to the next level with the apostrophes as well, nice!
https://github.com/gloc-mike/prj-rev-bwfs-dasmoto/blob/8d0c5497a09b92555333ab8d02b48cec8553a480/DasmotosArts/index.html#L4

**Comment by gloc-mike** [comment]
Yes, the ampersand caused some issues for the display of the site and I thought that the apostrophes might also.
